flyteplugins/go/tasks/logs/config.go

```diff
@@ -28,6 +28,9 @@ type LogConfig struct {
 	StackdriverLogResourceName string              `json:"stackdriver-logresourcename" pflag:",Name of the logresource in stackdriver"`
 	StackDriverTemplateURI     tasklog.TemplateURI `json:"stackdriver-template-uri" pflag:",Template Uri to use when building stackdriver log links"`
 
+	IsFlyinEnabled   bool                `json:"flyin-enabled" pflag:",Enable Log-links to flyin logs"`
+	FlyinTemplateURI tasklog.TemplateURI `json:"flyin-template-uri" pflag:",Template Uri to use when building flyin log links"`
+
```

Comment:
Why is this specific to flyin? This should be a map of string & tasklog.TemplateLogPlugin... task templates should be able to enable any number of them

Reply:
We sync'd offline and the idea is to make this more extensible by providing a way for tasks to dynamically set log links. I'll be sending a follow up PR shortly.